### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by ShadowJonathan** [mlemgroup/mlem#153]
Specifically, code testing

The test server we use is https://test-mlem.jo.wtf, however, if there's a very effective and integrated way to launch a docker container as part of the testing process (fully automatic) then please suggest this

This issue also leads into enabling CI for our workflows

**Comment by Sjmarf** [comment]
Related: #152

I'm personally against writing tests at scale whilst the team remains as small as it is now. Having to write tests would slow us down imo.

**Comment by EricBAndrews** [comment]
Agreed that they're not feasible right now. At some point this app will move into a low-velocity development pattern where having tests will be useful (e.g., for regression testing against Lemmy backend changes), but we're a long way off from that.

**Comment by Sjmarf** [comment]
Sounds good to me 👍
